provisioner/puppet-masterless/provisioner.go

```diff
@@ -73,7 +78,8 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 
 	// Set some defaults
 	if p.config.ExecuteCommand == "" {
-		p.config.ExecuteCommand = "{{.FacterVars}} {{if .Sudo}} sudo -E {{end}}" +
+		p.config.ExecuteCommand = "cd {{.WorkingDir}} && " +
```

Comment:
It seems like using `cd` here is not a very cross-platform (especially for Windows). I believe there is a way to set the working directory when we exec the command from Go.